app/views/users/_form.html.haml

```diff
@@ -0,0 +1,14 @@
+= form_for(@user) do |f|
+  %p First Name:
+  = f.text_field :first_name, :class => "form-control", required: true, autocomplete: "off"
```

Comment:
I tried to use this syntax always `:key => value` but it's mixed up in this line.